src/main/java/seedu/address/logic/commands/SuggestMateCommand.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.coursemate.CourseMate;
+import seedu.address.model.coursemate.HasRequiredSkillsPredicate;
+import seedu.address.model.coursemate.Name;
+import seedu.address.model.group.Group;
+import seedu.address.model.group.exceptions.GroupNotFoundException;
+import seedu.address.model.skill.Skill;
+
+/**
+ * Finds and lists all course mates in contact list whose skills match with uncompleted skills
+ * in the queried group. Course mates already in the group will not be listed.
+ */
+public class SuggestMateCommand extends Command {
+    public static final String COMMAND_WORD = "suggest-mate";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all course mates with any of "
+            + "the required skills from the given group and displays them as a list.\n"
+            + "The course mates already in the group are not displayed. groups can be specified by name.\n"
```

Comment:
Typo: "..displayed. groups can be specified by name"
groups => Groups
start with upper-case